utils/database.js

```diff
@@ -111,7 +107,9 @@ class Database {
               const row = results.rows.item(i);
 
               const gurmukhiLine =
-                visram && row.GurmukhiBisram ? row.GurmukhiBisram : row.Gurmukhi;
+                visram && row.GurmukhiBisram
+                  ? row.GurmukhiBisram
+                  : row.Gurmukhi;
```

Comment:
instead of using dot notation veerji, please use es6 syntax for destructing values from `row` object. Not only on this line but also at the places you are using dot notation with `row` variable in this function.

Reply:
Fixed